using-mobile-extensions/mobile-core/identity/identity-api-reference.md

```diff
@@ -26,7 +28,7 @@ Identity.syncIdentifier("idType", "idValue", VisitorID.AuthenticationState.AUTHE
 
 ### syncIdentifiers
 
-**Tip**: The `identifiers` map contains IDs with the Identifier type as the key, and the string identifier as the value.
+**Tip**: The `identifiers` map contains IDs with the Identifier type as the key, and the string identifier as the value. Both the identifier type and identifier should have valid string values (non null or empty), otherwise these identifiers are ignored by the Identity extension.
```

Comment:
Please reword as follows:

"The `identifiers` map contains IDs with the Identifier type as the key, and the string identifier as the value. If the identifier type and identifier do not have valid string values (non null or empty), the identifier is ignored by the Identity extension."